pom.xml

```diff
@@ -20,21 +20,21 @@
 
   <groupId>co.cask</groupId>
   <artifactId>whole-file-source</artifactId>
-  <version>1.1.1-SNAPSHOT</version>
+  <version>1.1.2-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <name>Whole file source</name>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <spark.version>1.6.1</spark.version>
-    <cdap.version>4.1.0</cdap.version>
+    <spark.version>2.1.3</spark.version>
```

Comment:
Does this plugin use Spark at all? If not, please remove the dependency.
If it does, just changing the spark version won't do the job as the artifact are different

Reply:
I think this plugin shouldn't depend on Spark at all.